extensions/ql-vscode/src/databases/db-item-expansion.ts

```diff
@@ -34,9 +34,9 @@ export interface RemoteUserDefinedListExpandedDbItem {
 export function updateExpandedItem(
   currentExpandedItems: ExpandedDbItem[],
   dbItem: DbItem,
-  itemExpanded: boolean,
+  addItem: boolean,
```

Comment:
This is minor, but I don't think this name is an improvement, mainly because the function will either add the item or remove it. I'd just split the function into two `addExpandedItem` and `removeExpandedItem`.